libraries/chain/wasm_interface.cpp

```diff
@@ -708,7 +708,15 @@ class producer_api : public context_aware_api {
          auto s = len * sizeof(chain::account_name);
          if( buffer_size == 0 ) return s;
 
-         auto copy_size = std::min( buffer_size, s );
```

Comment:
all **wasm_interface** methods accept data size parameter in raw _bytes_, not the number of objects. The function also returns the size in _bytes_.